JavaScript with Axios/pdf-with-added-image.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * This request demonstrates how to apply a file watermark to a PDF. This kind of watermark uses another PDF (using either watermark_file or watermark_file_id as a form-data parameter) and applies it as a watermark to the primary input document (file or id).
+ * Horizontal and vertical offsets of the watermark are measured in PDF units. (1 inch = 72 PDF units)
+ */
+var axios = require('axios');
+var FormData = require('form-data');
+var fs = require('fs');
+
+// Create a new form data instance and append the PDF file and parameters to it
+var data = new FormData();
+data.append('file', fs.createReadStream('/path/to/file'));
+data.append('watermark_file', fs.createReadStream('/path/to/file'));
+data.append('watermark_file_scale', '0.5');
+data.append('opacity', '0.5');
+data.append('x', '0');
+data.append('y', '0');
+data.append('rotation', '0');
+data.append('output', 'pdfrest_watermarked_pdf');
```

Comment:
These parameters aren't valid for `/pdf-with-added-image`.

Reply:
```suggestion
data.append('file', fs.createReadStream('/path/to/file'));
data.append('image_file', fs.createReadStream('/path/to/file'));
data.append('x', '72');
data.append('y', '72');
data.append('page', '1');
data.append('output', 'pdfrest_pdf_with_added_image');
```

Reply:
Thanks, are these from the Postman collection? Currently we have 

```
data.append('file', fs.createReadStream('/path/to/file'));
data.append('image_file', fs.createReadStream('/path/to/file'));
data.append('x', '0');
data.append('y', '0');
data.append('page', '1');
data.append('output', 'pdfrest_pdf_with_added_image');
```

Reply:
They're not really from anywhere per se; I only put in `72` because that equates to 1 inch.

Reply:
Alright I'll update these from what we have in the collection.

Reply:
The goal was to take the output from postman and use them as the code samples.